src/main/resources/schema.graphqls

```diff
@@ -33,6 +56,27 @@ input UserInput {
     password: String!
 }
 
+input FindUserInput {
+    username: String!
```

Comment:
planning to use some kind of session/token since you now no longer need to provide password ?